### PR TITLE
[Add] UserProfile

### LINKED ISSUE #10: User profile

### PR DESCRIPTION
**Pull request by gsc2001**
Fixes #10
Address component is not done will complete that later

### CODE REVIEW

File: components/Address/AddressCard.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            <Card.Content>
+                <Title>{address.name}</Title>
+                <Caption>
+                    {/* Hi */}
+                    <List.Item title={address.address1}></List.Item>
+                    <List.Item title={address.address2}></List.Item>
+                    <List.Item
+                        title={`${address.city}, ${address.state} ${address.pincode}`}
+                    ></List.Item>
+                    <List.Item title={address.country}></List.Item>
```
Comment (tichnas) [inline components/Address/AddressCard.tsx:21]: I don't think we need this :sweat_smile:

File: components/Address/AddressCard.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    address: Address;
+};
+
+const AddressCard: React.FC<AddressCardProps> = ({ address }) => {
+    return (
+        <Card elevation={3} focusable style={styles.card}>
+            <Card.Content>
+                <Title>{address.name}</Title>
+                <Caption>
+                    {/* Hi */}
```
Comment (tichnas) [inline components/Address/AddressCard.tsx]: remove this

File: components/Address/index.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import ViewPager from '@react-native-community/viewpager';
```
Comment (tichnas) [inline components/Address/index.tsx]: what's this used for?

File: features/Auth/userSlice.ts [hunk 4]
```diff
@@ -3,17 +3,17 @@ import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 export interface userState {
     token: string;
     isLoggedIn: boolean;
-    name: string;
-    phone: string;
-    email: string;
+    // name: string;
```
Comment (tichnas) [inline features/Auth/userSlice.ts]: remove these comments

File: components/Address/index.tsx (hunk 3 above)
Comment (gsc2001) [reply components/Address/index.tsx]: oh shit removing

File: components/Address/AddressCard.tsx (hunk 1 above)
Comment (gsc2001) [reply components/Address/AddressCard.tsx:21]: let it be there

File: components/Address/AddressCard.tsx (hunk 2 above)
Comment (gsc2001) [reply components/Address/AddressCard.tsx]: sure

File: features/Auth/userSlice.ts (hunk 4 above)
Comment (gsc2001) [reply features/Auth/userSlice.ts]: sure

Comment (gsc2001) [reply features/Auth/userSlice.ts:6]: i have kept them as in future we might need them so only uncomment it 😅

**Review by tichnas — APPROVED** [review]

**Pull request merged**